spec/std/io/memory_spec.cr

```diff
@@ -226,6 +226,16 @@ describe IO::Memory do
     io.gets(1).should eq("d")
   end
 
+  it "can seek and truncate" do
+    io = IO::Memory.new "hello world"
+    io.seek(-6, Seek::Current)
+    io.pos = 0
+    io.gets().should eq("hello")
```

Comment:
Please, stop using empty parentheses (ditto below)! they're anti-pattern in Crystal...

Reply:
I have tried to stop using my own style in PR's to Crystal. Sometimes I forget about some. I will correct it now.

Reply:
Ruby & Crystal hate needless `()` ;)